src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/interior_data_cache_test.cc

```diff
@@ -177,4 +178,21 @@ TEST_F(InteriorDataCacheTest, Exist2ModuleTypesRemoveOneAnotherOneLeft) {
   EXPECT_EQ(data2, retrieved_data2);
 }
 
+TEST_F(InteriorDataCacheTest, CheckCacheDataMerge) {
+  rc_rpc_plugin::InteriorDataCacheImpl cache;
```

Comment:
@YarikMamykin 
Move it to the place before its first usage

Reply:
@AByzhynar , fixed in 6886417